dev-requirements/mkdocs.txt

```diff
@@ -0,0 +1,23 @@
+mkdocs==1.5.3
```

Comment:
I don't think this is necessary, since mkdocs-material already depends on it, and it could cause versioning conflicts.

Reply:
Rather incompatibilities produce a error that we will catch when bumping dependencies than broken docs which are hard to fix